src/go/cmd/content.go

```diff
@@ -124,19 +125,37 @@ func (c *content) parseConst(tokenList *[]Token) {
 			}
 			makeToken(nil, nil, ")", punctuation, tokenList)
 			makeToken(nil, nil, "", 1, tokenList)
+
+			c.searchForPossibleValuesMethod(t, tokenList)
+			c.searchForMethods(t, tokenList)
 		}
 	}
 
 }
 
+func (c *content) searchForPossibleValuesMethod(t string, tokenList *[]Token) {
+	for i, f := range c.Funcs {
+		if i == fmt.Sprintf("Possible%sValues", t) {
+			makeFuncTokens(&i, f.Params, f.Returns, f.ReturnsNum, tokenList)
+			delete(c.Funcs, i)
+			return
+		}
+	}
+}
+
 // adds the specified function declaration to the exports list
 func (c *content) addFunc(pkg pkg, f *ast.FuncDecl) {
 	// create a method sig, for methods it's a combination of the receiver type
 	// with the function name e.g. "FooReceiver.Method", else just the function name.
 	sig := ""
 	if f.Recv != nil {
-		sig = "(c "
-		sig += pkg.getText(f.Recv.List[0].Type.Pos(), f.Recv.List[0].Type.End())
+		receiver := pkg.getText(f.Recv.List[0].Type.Pos(), f.Recv.List[0].Type.End())
+		if unicode.IsLower(rune(receiver[0])) || (string(receiver[0]) == "*" && unicode.IsLower(rune(receiver[1]))) {
```

Comment:
I think you want [`ast.IsExported`](https://pkg.go.dev/go/ast#IsExported) here.

Reply:
So `ast.IsExported` basically does a unicode.IsUpper underneath and from what I understood in the code it wouldnt accept receivers that have a pointer before the name...so the if would remain similarly large. I'd vote to keep it like this with the IsLower() directly being used.

Reply:
Can we write our own version to take this into account then?  `ast.IsExported` also includes properly decoding the rune which we should be doing.
```go
func IsExported(name string) bool {
	ch, _ := utf8.DecodeRuneInString(name)
	return unicode.IsUpper(ch)
}
```

Reply:
Done